app/views/spotlight/exhibits/index.html.erb

```diff
@@ -0,0 +1,51 @@
+<% if (current_user && current_user.exhibits.any?) || can?(:manage, Spotlight::Exhibit) %>
+  <ul class="nav nav-tabs" role="tablist">
+    <li role="presentation" class="nav-item"><a href="#published" aria-controls="published" role="tab" data-toggle="tab" class="nav-link active"><%= t('.published') %></a></li>
+    <% if can?(:manage, Spotlight::Exhibit) && @exhibits.unpublished.accessible_by(current_ability).any? %>
+      <li role="presentation" class="nav-item"><a href="#unpublished" aria-controls="unpublished" role="tab" data-toggle="tab" class="nav-link"><%= t('.unpublished') %></a></li>
+    <% end %>
+    <% if current_user && current_user.exhibits.any? %>
+      <li role="presentation" class="nav-item"><a href="#user" aria-controls="user" role="tab" data-toggle="tab" class="nav-link"><%= t('.user') %></a></li>
+    <% end %>
+  </ul>
+<% end %>
+
+<div class="tab-content">
+  <div role="tabpanel" class="tab-pane active" id="published">
+    <% if @exhibits.published.none? %>
+      <%= render 'missing_exhibits' %>
+    <% else %>
+      <%= render 'tags', tags: @exhibits.published.all_tags %>
+      <%= render 'exhibits', exhibits: @published_exhibits %>
+
+      <% if @published_exhibits.total_count > @published_exhibits.size %>
+        <nav>
+          <ul class="pagination">
+            <li class="page-link"><%= link_to_previous_page @published_exhibits, t('views.pagination.previous').html_safe %></li>
+            <li class="page-link"><%= link_to_next_page @published_exhibits, t('views.pagination.next').html_safe %></li>
+          </ul>
+        </nav>
+      <% end %>
+    <% end %>
+
+  </div>
+
+  <% if @exhibits.unpublished.accessible_by(current_ability).any? %>
+    <div role="tabpanel" class="tab-pane" id="unpublished">
+      <%= render 'exhibits', exhibits: @exhibits.unpublished.ordered_by_weight.accessible_by(current_ability) %>
+    </div>
+  <% end %>
+
+  <% if current_user && current_user.exhibits.any? %>
+    <div role="tabpanel" class="tab-pane" id="user">
+      <%= render 'exhibits', exhibits: current_user.exhibits %>
+    </div>
+  <% end %>
+</div>
+
+<% # Overridden just to override these classes %>
+<% content_for(:content_position) { 'order-12 order-lg-0' } %>
```

Comment:
Should we just do this in Spotlight?

Reply:
I think seeing something upstream in Spotlight would be interesting for this.  It certainly doesn't work super well as-is, so any improvement would be 👍 from me.